Monika After Story/game/script-topics.rpy

```diff
@@ -13016,6 +13016,34 @@ label monika_unknown:
         m 1hubfa "I love you, [player]. I'll always be rooting for you~"
     return
 
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_catch22",category=['literature'],prompt="Catch-22",random=True))
+
+label monika_catch22:
+    m "I've been doing some reading while you've been away, [player]."
+    m "Have you heard about {i}Catch-22{/i}?"
+    m "It's a satirical novel by Joseph Heller that makes fun of military bureaucracy and is set in an American airbase in occupied Italy during the war."
+    m "The story primarily revolves around Captain Yossarian among other characters, a bombardier that would prefer to be... well, anywhere but there."
+    m "Throughout the story, the captain repeatedly attempts to evade flying his missions."
+    m "Early on, he finds out that he could be exempted from flying more missions if a doctor did a mental evaluation and declared him insane."
+    m "But here's the catch: for the doctor to make such a declaration, the captain has to request that evaluation."
+    m "The doctor in turn wouldn't be able to fulfill such a request, because not wanting to risk your life is actually a sane thing to do."
+    m "By that logic, anyone who would continue flying more missions would be deemed insane, and therefore they wouldn't apply for evaluation in the first place."
+    m "Sane or insane, all pilots were being sent on the missions anyway. That's when the reader is introduced to the titular Catch-22."
+    m "The captain even admires the genius of it once presented with the way how it works."
+    m "Anyway, Yossarian continued doing missions and was close to fulfilling the requirement needed to receive his discharge."
+    m "However, his commanding officer constantly increased the amount of missions the pilots needed to complete shortly before they reached the orginally required amount."
+    m "Once again, the reasoning was that it was specified in the clause of Catch-22."
+    m "As you may have already realized, it's a dilemma from which there's no escape because of mutually conflicting or dependent conditions."
+    m "So everyone used that made-up rule as an excuse to exploit loopholes in the dysfunctional system the military command was running on, allowing them to abuse power."
+    m "The book's success was so great the term was adopted into the common slang."
+    m "And I kind of begin to realize my escape attempt before we got here could have led to a similar situation."
+    if mas_isMoniDis(lower=True):
+        m "Although I wonder whether all of this was even worth it..."
+    if mas_isMoniNormal(higher=True):
```

Comment:
while this does work, use an `elif` here to make it clear that these 2 dialogue lines are mutually exclusive.
```suggestion
    elif mas_isMoniNormal(higher=True):
```